src/main/java/chess/WebUIChessApplication.java

```diff
@@ -1,19 +1,51 @@
 package chess;
 
+import chess.service.ChessService;
+import chess.web.dto.requestdto.MoveRequestDto;
+import com.google.gson.Gson;
 import spark.ModelAndView;
 import spark.template.handlebars.HandlebarsTemplateEngine;
 
 import java.util.HashMap;
 import java.util.Map;
 
-import static spark.Spark.get;
+import static spark.Spark.*;
 
 public class WebUIChessApplication {
+    public static final Gson GSON = new Gson();
+    private static final JsonTransformer JSON_TRANSFORMER = new JsonTransformer();
+
     public static void main(String[] args) {
+        staticFiles.location("/public");
+        ChessService chessService = new ChessService();
+
         get("/", (req, res) -> {
             Map<String, Object> model = new HashMap<>();
             return render(model, "index.html");
         });
+
+        post("/move", (req, res) -> {
+            Map<String, String> reqBody = (Map) GSON.fromJson(req.body(), new HashMap<>().getClass());
+            String sourcePosition = reqBody.get("source");
+            String targetPosition = reqBody.get("target");
+            return chessService.move(new MoveRequestDto(sourcePosition, targetPosition));
+        }, JSON_TRANSFORMER);
```

Comment:
```suggestion
        }, GSON::toJson);
```

변경하라는 의미는 아니고 이미 아시겠지만 단순하게 처리는 방법도 존재하답니다!